### PR TITLE
Use query parameter for pages

### PR DESCRIPTION
**Pull request by andreas-aeschlimann**
This pull requests solves the navigations of the application. If you accept it and merge it, the code will be automatically added to your repository.

Look at the following code in the JavaScript file:

https://github.com/JanisTh/gkg-inf-project/blob/a06db35881d5c649c9af2bae59e94890a6931da2/script.js#L22-L55

If you look at line 32, you see that we ask for an URL param named `page`. An URL param is the part of the URL after the question mark, for example `https://page.com/index.html?page=home`. In this example, we have a query param `page` with the value `home`.

Line 36 includes the page (for example `home`) from the `pageDirectory` (as defined on line 7, the directory is named `pages`). Hence, if you want to include your pages as intended, you have to fix all URLs in your application to this schema.

TLDR; Do not make links to `pages/home.html`. Instead, make links to `?page=home`.

### CODE REVIEW

**Review by andreas-aeschlimann — COMMENTED** [review]
.

File: menu.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-<button id="teamButton" class="navigationButtons" onclick="window.location.href = 'page/zimmer.html'">Zimmer</button>
-<button id="hireUsButton" class="navigationButtons" onclick="window.location.href = 'page/buchen.html'">Buchen</button>
-<button id="imprintButton" class="navigationButtons" onclick="window.location.href = 'page/ueberUns.html'">Über uns</button>
-<button id="kontaktButton" class="navigationButtons" onclick="window.location.href = 'page/kontakt.html'">Kontakt</button>
\ No newline at end of file
+<button id="homeButton" class="navigationButtons" onclick="window.location.href = '?page=home'">Home</button>
+<button id="teamButton" class="navigationButtons" onclick="window.location.href = '?page=zimmer'">Zimmer</button>
+<button id="hireUsButton" class="navigationButtons" onclick="window.location.href = '?page=buchen'">Buchen</button>
+<button id="imprintButton" class="navigationButtons" onclick="window.location.href = '?page=ueberUns'">Über uns</button>
+<button id="kontaktButton" class="navigationButtons" onclick="window.location.href = '?page=kontakt'">Kontakt</button>
```
Comment (andreas-aeschlimann) [inline menu.html]: See here how I changed the URLs. This needs to be done in the whole application.

**Review by Soraturtle09 — COMMENTED** [review]
It works now, thank you for explaining it to us.
